Makefile.am

```diff
@@ -241,39 +241,12 @@ endef
 # Some of our code is arranged into private libraries used during
 # build, but not published or distributed outside the build-tree.
 
-noinst_LTLIBRARIES += 		libpriv-ic.la
-
-libpriv_ic_la_SOURCES = \
-					src/shared/ic.h \
-					src/shared/ic.c
-
 noinst_LTLIBRARIES += 		libpriv-tntnet-mlm.la
 
 libpriv_tntnet_mlm_la_SOURCES = \
 					src/shared/tntmlm.h \
 					src/shared/tntmlm.cc
 
-noinst_LTLIBRARIES += 		libpriv-utils-web.la
-
-libpriv_utils_web_la_SOURCES = \
-				src/shared/utils_web.h \
-				src/shared/utils_web.cc
-
-noinst_LTLIBRARIES += 		libpriv-utils-plusplus.la
-
-libpriv_utils_plusplus_la_SOURCES = \
-				src/shared/utils++.cc \
-				src/shared/utils++.h \
-				src/shared/filesystem.h \
-				src/shared/filesystem.cc
-
-libpriv_utils_plusplus_la_LIBADD = ${LIBCZMQ_LIBS}
```

Comment:
IIRC This was the bit required for clean linking - add it to the newly surviving lib recipe